### DISCUSSION TITLE
I believe my obligations end here (for now)

### DISCUSSION THREAD

**Discussion by nashmuhandes** [Realm667/Re-Releases#269] [General]
@Tormentor667 @Blue-Shadow I would like to formally announce that I think I've done everything I could (and wanted -- given the time limit) for the project.

It was a good experience working on this and I think the result is spectacular!

I will hang around for a bit and be on standby in case I need to fix anything that got broken unintentionally, but what this post generally means is:

- I will no longer be editing the map .wad
- I won't be adding any more new content
- I won't be doing any more editing or tweaking (unfortunately, Blue Shadow, this also means that I don't think l'm equipped to do anything about the Doom Thamuz - looking back at it, I didn't really do much when it came to the boss in general, and I don't have the resources to address anything missing for it currently, anyway...)

If there are any last minute things I need to fix, feel free to assign me in the Issues tab, but otherwise, this is where I call it, I think. :)

As per discussion with @Tormentor667 the other day; from this point on, I will be preparing to make some content related to the social media aspects of the project.

**Comment by Tormentor667** [comment]
Thanks a lot to you for your dedication - same to @Blue-Shadow and @Ozymandias81 as well. This remaster is gonna be great, I am already passing it to friends and I am getting a lot of positive feedback :)
